frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -532,28 +545,34 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
       castAndPrimLength = this.segmentToCast.get(castUUID);
     }
     this.primaryGroupNum = castAndPrimLength[1];
-    this.castDnD ? this.castDnD.selectCast({uuid: castUUID, saveDeleteEnabled: false}) : '';
+    if (this.castDnD) {
+      this.castDnD.selectCast({uuid: castUUID, saveDeleteEnabled: false});
```

Comment:
A better way to write this would be to get rid of the `if (this.castDnD)` and just have:
`this.castDnD?.selectCast({uuid: castUUID, saveDeleteEnabled: false});`

Reply:
By the way, this applies to several sections in the current PR

Reply:
Done.